lib/ansible/plugins/doc_fragments/zabbix.py

```diff
@@ -48,4 +48,7 @@ class ModuleDocFragment(object):
       type: bool
       default: yes
       version_added: "2.5"
+notes:
+    - If you use I(login_password=zabbix), the word "zabbix" is replaced by "********" in all module output, because I(login_password) uses C(no_log).
+      See L(this FAQ,https://docs.ansible.com/ansible/devel/network/user_guide/faq.html#why-is-my-output-sometimes-replaced-with) for more information.
```

Comment:
My bad, I pointed you to `devel` when all links should point to `latest`:
```suggestion
      See L(this FAQ,https://docs.ansible.com/ansible/latest/network/user_guide/faq.html#why-is-my-output-sometimes-replaced-with) for more information.
```